src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Assign Emission Admin - Ethereum, Arbitirum and Optimism"
+author: "karpatkey-TokenLogic & ACI"
```

Comment:
you must add a discussion link as i think uis can't handle it missing. snapshot you can leave empty.